src/activities/Association.js

```diff
@@ -1,12 +1,11 @@
 import { Entity } from '../es6/Entity.js';
-import { Actions, Rels } from '../hypermedia-constants';
+import { Actions, Classes, Rels } from '../hypermedia-constants';
 import { performSirenAction } from '../es6/SirenAction';
-
 /**
  * Association class representation of an association.
  */
 
-export class Association extends Entity {
+export class AssociationEntity extends Entity {

```

Comment:
Can you also `export const Association = AssociationEntity;` so it doesn't break? multi-repo madness and all...